### PR TITLE
fix: use AppConfig to configure logging for server

### PR DESCRIPTION
**Pull request by Bento007**
- add comments to describe the connection between verbose and debug in the CLI and the default config
- configure_logging to across all server code.
- update default values for CLI so verbose and debug can be toggled with flags.

### CODE REVIEW

File: server/cli/launch.py [hunk 1]
```diff
@@ -269,9 +266,6 @@ def launch(
     # create the server
     server: TestServer = TestServer(app_config)
 
-    if not app_config.server__app__verbose:
-        log.setLevel(ERROR)
-
```
Comment (seve) [inline server/cli/launch.py:274]: why was this line not working? Was it because it was being overridden by `configure_loader`

Comment (Bento007) [reply server/cli/launch.py:274]: yes, it was being overridden by `configure_logging`

File: server/cli/launch.py [hunk 2]
```diff
@@ -68,16 +67,16 @@ def server_args(func):
...
         help="Run in debug mode. This is helpful for cellxgene developers, "
-        "or when you want more information about an error condition.",
+        "or when you want more information about an error condition. This will automatically set verbose mode.",
     )
     @click.option(
         "--verbose",
         "-v",
         is_flag=True,
-        default=DEFAULT_CONFIG.server__app__verbose,
+        default=False,
```
Comment (Bento007) [inline server/cli/launch.py:79]: the default should be false unless the flag is specified on the CLI

File: server/cli/launch.py [hunk 3]
```diff
@@ -68,16 +67,16 @@ def server_args(func):
         "--debug",
         "-d",
         is_flag=True,
-        default=DEFAULT_CONFIG.server__app__debug,
+        default=False,
```
Comment (Bento007) [inline server/cli/launch.py:70]: the default should be false unless the flag is specified on the CLI

File: client/__tests__/e2e/test_config.yaml [hunk 4]
```diff
@@ -2,8 +2,8 @@ server:
   app:
     force_https: true
     port: 5005
-    debug: true
-    verbose: true
+    debug: false
```
Comment (Bento007) [inline client/__tests__/e2e/test_config.yaml:5]: if debug is true then verbose will be set to true. They must both be false to disable verbose

File: server/ecs/app.py [hunk 5]
```diff
@@ -13,28 +12,9 @@
 SERVERDIR = os.path.dirname(os.path.realpath(__file__))
 sys.path.append(SERVERDIR)
 
-
-dictConfig(
```
Comment (Bento007) [inline server/ecs/app.py:17]: consolidating log setup into `configure_logging()`

**Comment by seve** [comment]
Pulled this locally and getting this output:
```sh
./launch_dev_server.sh --config-file ./client/__tests__/e2e/test_config.yaml --port 5005
[cellxgene] Starting the development server...
[cellxgene] Launching! Please go to http://localhost:5005 in your browser.
[cellxgene] Type CTRL-C at any time to exit.
 * Serving Flask app 'server.app.app'
 * Debug mode: on
```

**Review by seve — APPROVED** [review]

**Pull request merged**